spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -993,6 +993,7 @@ public String getSparkUIUrl() {
   }
 
   private Results.Result interpret(String line) {
+    out.ignoreLeadingNewLinesFromScalaReporter();
```

Comment:
is there a reason we have to set this every time interpret is called, and not have it sticky?